pkg/auth/dependency/webapp/state.go

```diff
@@ -1,13 +1,32 @@
 package webapp
 
 import (
+	"net/url"
+
 	"github.com/authgear/authgear-server/pkg/core/skyerr"
 )
 
 type State struct {
-	ID              string           `json:"id"`
+	FlowID          string           `json:"flow_id"`
+	InstanceID      string           `json:"instance_id"`
```

Comment:
Do we need these two fields?

Reply:
Seems unnecessary. Let me remove FlowID first.